generation_config.yaml

```diff
@@ -789,12 +790,13 @@ libraries:
   - proto_path: google/cloud/discoveryengine/v1alpha
   - proto_path: google/cloud/discoveryengine/v1beta
 
-- api_shortname: distributedcloudedge
+- api_shortname: edgecontainer
```

Comment:
`api_shortname` is used to generate the folder name, to keep the folder name the same, please add the following
```
library_name: distributedcloudedge
```

Reply:
`api_shortname` is supposed to match the service's subdomain. Metrics are looking at the `api_id` and falling back to `api_shortname` + `.googleapis.com` if `api_id` is not present.

Would setting `library_name` be more appropriate?

Reply:
Looks like `managedidentities` has `api_shortname: managedidentites` and `library_name: managed-identities` and it uses `java-managed-identities` as the folder name.